pyproject.toml

```
@@ -0,0 +1,3 @@
+[build-system]
+requires = [ "setuptools >= 35.0.2", "wheel >= 0.29.0"]
```

I think I copied them from somewhere else, lets get the best versions that py2.7 supports:

```suggestion
requires = [ "setuptools >= 44.1.1", "wheel >= 0.36.2"]
```